docs/DEVELOPMENT.md

```diff
@@ -35,6 +34,34 @@ npx tauri dev
 
 Automatic recompiling on save is enabled for both the Rust and the Typescript/Vue code.
 
+If you want to build FlightCore from source code, do the first two commands and:
+
+```sh
+cd ../src-tauri
+```
+
+```sh
+npm run tauri build
```

Comment:
I would shorten it to just:

```suggestion
If you want to build FlightCore from source, run

```sh
npx tauri build
```

Reply:
Will this also take care of npm and vue dependencies?

Reply:
No. The way I read it it's a continuation of the section above, so like run `npx tauri dev` for dev setup, `npx tauri build` for release build.

Reply:
Doesn't `cd src-vue && npm install` imply that build should be run in src-vue, instead of the main folder?

Reply:
Kinda but then it needs to be fixed in the section above cause you need to do the same things for `npx tauri dev` and `npx tauri build`.

Given slightly altered instructions for one but not for the other just makes the whole thing more confusing IMO.

Reply:
cd .. && npx tauri dev

Reply:
> cd .. && npx tauri dev or [cd src-vue && npm install && cd ..](https://github.com/R2NorthstarTools/FlightCore/blob/main/.github/workflows/release.yml#L51)

Essentially it's 
```
npm install
cd src-vue
npm install
cd ..
```
to install node dependencies and then one can run `npx tauri dev` or `npx tauri build`

&nbsp;

>  By the way, will you also change to npx in the workflow file?

Eh idc, `npx` is just `npm run` AFAIK so doesn't really matter for scripting etc. For instructions however it should be consistent though. So either always `npx` or always `npm run`